src/main/java/org/jitsi/videobridge/util/ByteBufferPool.java

```diff
@@ -113,6 +127,19 @@ private static long threadId()
         return Thread.currentThread().getStackTrace();
     }
 
+    /**
+     * Gets a stack trace as a multi-line string.
+     */
+    private static String stackTraceAsString(StackTraceElement[] stack)
```

Comment:
While you're doing this, can you write a kotlin one which is inlined so we don't get these methods as part of the stack?  I'm fine with just doing it in JVB.

Reply:
I can use `UtilKt.getStackTrace()` but Java won't inline Kotlin, so they still show up on the stack.  Probably still worth it, though.

Reply:
Ah, bummer.